core/src/main/scala/formula/Form.scala

```diff
@@ -140,10 +139,10 @@ object Form {
             acc.zip(v.get).map { case (acc, v) => acc.appended(v) }
           }
 
-        override def set(a: List[A]): Unit =
-          a.zip(variables.now()).foreach { case (a, v) =>
-            v.set(a)
-          }
+        override def set(a: List[A]): Unit = {
+          countForm.set(a.length)
+          setTimeout(() => a.zip(variables.now()).foreach { case (a, v) => v.set(a) }, 50)
```

Comment:
> Add a short delay when loading values for Many form fields since variables.now() may not be up to date yet after updating the count field

Not very happy about the solution here, pls let me know if you have suggestions on how to improve this.

Reply:
I just noticed that the same problem occurrs for `Either` and `Option` forms, but I think it's only happening in certain cases, i.e. for a more complex form in my project but not in the examples.

I'll try fix those as well; either here if you want to wait or in an upcoming PR.

Reply:
I'm happy to merge now and add additional PRs on top :)